paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/CardDetailsUI.kt

```diff
@@ -152,10 +193,24 @@ private fun CardNumberField(
     shouldShowCardBrandDropdown: Boolean,
     savedPaymentMethodIcon: Int,
     onBrandChoiceChanged: (CardBrandChoice) -> Unit,
+    isFirstField: Boolean,
 ) {
     CommonTextField(
         value = "•••• •••• •••• ${last4 ?: "••••"}",
         label = stringResource(id = R.string.stripe_acc_label_card_number),
+        shape = if (isFirstField) {
```

Comment:
Why is this now necessary?

Reply:
because we're rendering the name field on top of the card section. When name is collected, the card field does not have rounded borders, but it does if it's the first field (no name collected)